binding/springws/pom.xml

```diff
@@ -81,6 +76,14 @@
 			<groupId>io.tracee</groupId>
 			<artifactId>tracee-testhelper</artifactId>
 		</dependency>
+		<!--This dependency is needed for the integration test with jetty. The Spring servlet uses ServletRequestAttributes which tries to
+        find NumberUtils.STANDARD_NUMBER_TYPES. They exists only in spring-core 4.1.x but 4.0 is pulled by spring-ws.-->
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-core</artifactId>
+			<version>${spring.version}</version>
+			<scope>provided</scope>
```

Comment:
just wondering, why is it provided then? This will pull it on the compiler classpath (and we'd also compile against 4.1 and its not only relevant for tests).